telos-frontend/src/reportWebVitals.js

```diff
@@ -1,4 +1,4 @@
-const reportWebVitals = onPerfEntry => {
+const reportWebVitals = (onPerfEntry) => {
```

Comment:
I see that the added tools changed this to conform to the correct coding convention.